tests/unit/Converters/Base/FieldTest.php

```diff
@@ -153,9 +155,9 @@ private function getMockFieldDefinition(FieldModel $mockField)
      * @param int $fieldId
      * @param int $groupId
      *
-     * @return Mock|FieldModel
```

Comment:
a mock object is also instance of the thing it mocks...

Reply:
yeah I got that so I reverted it ;-)